kubernetes/samples/charts/ingress-per-domain/values.yaml

```diff
@@ -25,3 +25,7 @@ voyager:
   webPort: 30305
   # stats port
   statsPort: 30315
+
+nginx:
```

Comment:
Add comment above line 29: "# NGINX specific values"

Reply:
done

Reply:
i refreshed, but don't see the new comment - did you push the change?